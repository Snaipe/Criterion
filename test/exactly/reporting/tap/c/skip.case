[act]

% skip.c.bin --tap

[assert]

stdout
  -transformed-by replace_version
  equals -contents-of expected/skip.txt
