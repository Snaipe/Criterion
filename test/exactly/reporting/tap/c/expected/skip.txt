TAP version 13
1..2
# Criterion <VERSION>
#
# Running 2 tests from misc
ok - misc::skipping  # SKIP test was skipped
ok - misc::message  # SKIP Skips may take printf-like messages
